db/migrate/20210306104224_add_category_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddCategoryToArticles < ActiveRecord::Migration[6.1]
+  def change
+    add_column :articles, :category, :string
```

Comment:
I would have made the category as an enum, to have set values like we discussed earlier